packages/ketcher-core/src/domain/entities/sgroupForest.ts

```diff
@@ -184,3 +185,18 @@ export function checkOverlapping(struct, atoms) {
       : atoms.findIndex((aid) => sgAtoms.indexOf(aid) === -1) >= 0;
   });
 }
+
+export function checkQuerySGroupOverlapping(struct: Struct, atoms: number[]) {
```

Comment:
Can we use full names instead of abbreviations in this function?

Reply:
done